homeassistant/components/airzone_cloud/climate.py

```diff
@@ -144,8 +144,13 @@ class AirzoneClimate(AirzoneEntity, ClimateEntity):
     """Define an Airzone Cloud climate."""
 
     _attr_name = None
-    _attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
+    _attr_supported_features = (
+        ClimateEntityFeature.TARGET_TEMPERATURE
+        | ClimateEntityFeature.TURN_OFF
+        | ClimateEntityFeature.TURN_ON
```

Comment:
> How can this be set at this point, while their child classes determine the available HAVC modes?

`AirzoneDeviceClimate` and `AirzoneDeviceGroupClimate` both define `async_turn_on` and `async_turn_off` because all Airzone devices have a separate on/off status from their HVAC mode.
This change is correct since all Airzone Climate classes inherit those two classes, so all of them support on/off.

Reply:
Right, but they are not implemented in this base class, nor is there anything in place that ensures that (no abstract). So, in that case, this should move to their child classes instead?

Reply:
> Right, but they are not implemented in this base class, nor is there anything in place that ensures that (no abstract). So, in that case, this should move to their child classes instead?

That's true, those methods aren't defined in the base AirzoneClimate class, so I guess that we could add:
```
    async def async_turn_on(self) -> None:
        raise NotImplementedError

    async def async_turn_off(self) -> None:
        raise NotImplementedError
```
To the base AirzoneClimate class. I can create a new PR with that change if you want :)

Reply:
Should move to the child classes. Let me know if you take care of that @Noltari or I'll do it tonight

Reply:
Should the existing `ClimateEntityFeature.TARGET_TEMPERATURE` flag also move into the child classes then?
`async_set_temperature` is only overridden in (all) child classes too.

Reply:
Thanks for the PR @TheJulianJES and sorry for my late answer @gjohansson-ST, but it has been a busy weekend...